app/marketing/models.py

```diff
@@ -80,8 +80,8 @@ def github_url(self):
         return f"https://github.com/{self.github_username}"
 
     @property
-    def local_avatar_url(self):
-        return f"/funding/avatar?repo={self.github_url}&v=3"
+    def avatar_url(self):
+        return f"/static/avatar/{self.github_username}"
```

Comment:
We could probably switch the avatar field to an image field on Profile and just pull down the avatar once and drop it into S3 storage and serve from behind CF.

Reply:
lets maybe make a v2 task for this?

Reply:
That sounds good to me!